### DISCUSSION TITLE
How to change TreeView's expand button color?

### DISCUSSION THREAD

**Discussion by YoriMirus** [AvaloniaUI/Avalonia#7959] [Q&A] [answered]
Hello. I have a TreeView which I want to use on a dark background. I have everything set up but I don't know how to set the colour of the expand button. By default it's black so it's hard to see. How can I change its colour?

**Comment by timunie** [comment, marked as answer]
Hi @YoriMirus 

I think you can override this Resource in your App: https://github.com/AvaloniaUI/Avalonia/blob/0551408203c460b6f6c83bccebf6704552342dd9/src/Avalonia.Themes.Fluent/Controls/TreeViewItem.xaml#L46

Happy coding 
Tim

**Reply by YoriMirus** [reply]
Thank you, that worked.
For anyone wondering how I did it, since I had multiple TreeViews, I just added the color as a resource to the Grid they belonged to (thus overriding it)
Example:

`
<Grid>
    <Grid.Resources>
        <SolidColorBrush x:Key="TreeViewItemForeground"  Color="White"/>
    </Grid.Resources>
</Grid>
`
